xbmc/utils/ScraperUrl.cpp

```diff
@@ -339,6 +339,15 @@ bool CScraperUrl::ParseEpisodeGuide(std::string strUrls)
   return true;
 }
 
+bool CScraperUrl::AddElement(std::string url, std::string aspect /* = "" */)
+{
+  TiXmlElement thumb("thumb");
```

Comment:
I think it´s better to directly create an SUrlEntry  object here and append it to m_url instead of goin through xml.